indico/modules/events/contributions/lists.py

```diff
@@ -49,13 +54,52 @@ def __init__(self, event):
                      'filter_choices': type_empty | type_choices},
             'status': {'title': _('Status'), 'filter_choices': {'scheduled': _('Scheduled'),
                                                                 'unscheduled': _('Not scheduled')}},
+        }
+
+        self.list_config = self._get_config()
+        self.extra_filters = {
             'people': {'title': _('People'), 'filter_choices': {'registered': _('Registered'),
                                                                 'not_registered': _('Nobody registered')}},
             'speakers': {'title': _('Speakers'), 'filter_choices': {'registered': _('Registered'),
                                                                     'not_registered': _('Nobody registered')}},
         }
 
-        self.list_config = self._get_config()
+    def _get_static_columns(self, ids):
+        """
+        Retrieve information needed for the header of the static columns.
+
+        :return: a list of {'id': ..., 'caption': ...} dicts
+        """
+        return [{'id': id_, 'caption': self.static_items[id_]['title']} for id_ in self.static_items if id_ in ids]
```

Comment:
I think they are not necessarily a subset of static items since the function separating the ids takes as dynamic any item that has an id that is not a `str`